clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -1,5 +1,5 @@
 /* **********************************************************
```

Comment:
> There is likely something missing in retaddr stack tracking, which doesn't seem to anticipate that function markers may be added without a prior call instruction. This needs to be filed separately (TODO sharmaabhinav: file an issue). Also separately, maybe we don't need to track retaddrs in a stack, as we seem to check only the last one (when we see the function return address marker soon after the call).

I thought there was tailcall handling; and as noted that typically requires a stack and the current stack pointer.

Reply:
Yes I realized the tail call part after the offline discussion. I've added comments to the invariant checker to note this more explicitly